dev/ctia/dev/split_tests.clj

```diff
@@ -1,7 +1,10 @@
 (ns ctia.dev.split-tests
   (:require [circleci.test :as t]
+            [clojure.data.priority-map :refer [priority-map-keyfn]]
```

Comment:
just discovered these collections.
I like it :-)

Reply:
Yes this saved my butt.